File: LeetCode/1281. Subtract the Product and Sum of Digits of an Integer/solution.go
/**
 * Runtime: 0 ms
 * Memory Usage: 1.8 MB 
 */
 func subtractProductAndSum(n int) int {
    p, s := 1, 0
    for ; n > 0; n /= 10 {
        p *= n % 10
        s += n % 10
    }
    return p - s
}

File: LeetCode/1281. Subtract the Product and Sum of Digits of an Integer/Solution.java
/**
 * Runtime: 0 ms
 * Memory Usage: 38.1 MB 
 */
class Solution {
    public int subtractProductAndSum(int n) {
        int p = 1, s = 0;
        while (n > 0) {
            p *= n % 10;
            s += n % 10;
            n /= 10;
        }
        return p - s;
    }
}
